rebar.config

```diff
@@ -8,7 +8,7 @@
 {xref_checks, [undefined_function_calls,locals_not_used]}.
 {profiles, [
   {docs, [
-    {deps,      [{edown, "0.8.1"}]},
+    {deps,      [{edown, "0.9.1"}]},
```

Comment:
It's possible this was used in the past: I can't make it work easily, at the moment, but it'd be even better to replace doc with `ex_doc`. I'll create an issue for this.

Reply:
https://github.com/elli-lib/elli/issues/121

Reply:
This is actually already done in another pull request: https://github.com/elli-lib/elli/pull/112